src/controller/model/endpoint.ts

```diff
@@ -202,12 +205,13 @@ class Endpoint extends Entity {
         debug.info(log);
 
         try {
+            transactionSequenceNumber = transactionSequenceNumber || ZclTransactionSequenceNumber.next();
             const frame = Zcl.ZclFrame.create(
```

Comment:
The options should also be honoured in all other method, like `read`, `command`, etc.

Reply:
Updated